CHANGELOG.rst

```diff
@@ -19,6 +19,10 @@ Unreleased
   since some providers do not has that step in the onboarding process.
 * Changed handler for exam ping to remove learner from the exam on 403 error.
 
+[2.5.5] - 2020-12-22
```

Comment:
Since you're proposing a new release all unreleased data should be added to the release as well